notebooks/curation_api/README.md

```diff
@@ -0,0 +1,24 @@
+# Discover API notebooks
+
+The Discover API notebooks are pre-baked scripts to facilitate use of the 
+[Discover API](https://api.cellxgene.cziscience.com/curation/ui/#/), which itself supports 
+curation flow interactions with the [CZ CELLxGENE Discover](https://cellxgene.cziscience.com/) tool.
+
+## Notebook usage
+
+Many notebooks have a few fields that require user input, such as the relevant Collection id or Dataset id.
+
+### Python notebooks: `python/`
+
+These notebooks are the most user-friendly; start here if you're unsure. All access token generation, url construction, and response 
+handling is abstracted away from the user. Unitary interactions such as getting a Dataset, creating a Collection, etc., 
+are each handled by single function calls.
+
+### "Raw" Python notebooks: `python_raw/`
+
+These notebooks show the construction of http requests at a more granular level, and as such may be useful for technical
+users.
+
+### R notebooks: `R/`
+
+These notebooks are the same as the "raw" Python notebooks mentioned immediately previous, but in R.
```

Comment:
It seems like this directory should be named R_raw to be symmetrical?

Reply:
Renamed (see diff below)

Reply:
I can also move the entire `notebooks/curation_api/` dir to `notebooks/discover_api/` in a follow-up PR...